identity-account-storage/src/storage/memstore.rs

```diff
@@ -247,3 +247,70 @@ impl Default for MemStore {
     Self::new()
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use crate::storage::Storage;
+
+  use super::MemStore;
+
+  fn test_memstore() -> Box<dyn Storage> {
+    Box::new(MemStore::new())
+  }
```

Comment:
Yes, our stronghold wrapper cannot be tested concurrently (there is an issue _somewhere_), so only with passing `--test-threads=1` to the test binary, and I don't think we can easily change that in CI for just `identity-account-storage` because we just run `cargo test` in the root, afaik. I ran the test suite locally and it passed for stronghold. Given that #787 is only waiting for some smaller changes from the stronghold team before it can be merged, that seems fine to me.